twitter.js

```diff
@@ -40,10 +44,17 @@ twitter.stream('statuses/filter', {'locations':'-180,-90,180,90'}, function (str
           });
           // printing surrogate pairs
           surrogate.forEach((surrogate) => {
+            if(emojiList[surrogate]){
+              codeTweets.push(emojiList[surrogate].name);
+            }
             var surrogatePair = surrogate.split('\\u').slice(1);
             var code = '0x';
             // console.log(String.fromCharCode(code+surrogatePair[0], code+surrogatePair[1]));
           });
+          if(codeTweets.length !== 0){
+            countryTweet['text'] = codeTweets;
```

Comment:
This is a little confusing, is `countryTweet.text` supposed to be a string, or an array? The name suggests it should be a string.

`codeTweets` is initialized to an empty array, so if it has no length `countryTweet.text` is being set to an empty array?